src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/FundingModal/TokenItem.tsx

```diff
@@ -0,0 +1,46 @@
+import { toFinite } from 'lodash';
+import React, { type FC } from 'react';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import Numeral from '~shared/Numeral/Numeral.tsx';
+import TokenIcon from '~shared/TokenIcon/TokenIcon.tsx';
+import { formatText } from '~utils/intl.ts';
+import { getSelectedToken } from '~utils/tokens.ts';
+
+import { type TokenItemProps } from './types.ts';
+
+const TokenItem: FC<TokenItemProps> = ({
+  amount,
+  networkFee,
+  tokenAddress,
+}) => {
+  const { colony } = useColonyContext();
+  const token = getSelectedToken(colony, tokenAddress);
+
+  return (
+    <div className="flex items-center justify-between gap-2">
+      {token && (
+        <div>
+          <div className="flex items-center gap-1 text-3">
+            <TokenIcon token={token} size="xxs" />
+            {token.name}
+          </div>
+          <span className="mt-1 text-xs text-gray-600">
+            {formatText({ id: 'fundingModal.fee' })}
+          </span>
+        </div>
+      )}
+      <div className="flex flex-col items-end gap-1">
+        <span className="text-3">
+          <Numeral value={amount} decimals={token?.decimals} /> {token?.symbol}
+        </span>
+        <span className="text-xs text-gray-600">
+          <Numeral value={toFinite(networkFee)} decimals={token?.decimals} />{' '}
```

Comment:
Why do we need `toFinite`?

Reply:
I saw one of Armando's PR recently and I thought it would also be needed here because of decimal numbers, but from what I see it also works well without it.